common/test/acceptance/pages/studio/overview.py

```diff
@@ -845,6 +845,15 @@ def deprecated_advance_modules(self):
         return self.q(css='.advance-modules-list li').text
 
 
+class CourseOutlineSignInRedirectPage(CourseOutlinePage):
```

Comment:
@jzoldak I understand your concern about the login. I wonder once user gets logged in, then it should be redirected to the course page. If we only inherit it from LoginPage then how should we accomplish this? Are you suggesting some design pattern which I don't know of?